data/hooks/conf_regen/43-dnsmasq

```diff
@@ -6,6 +6,9 @@ set -e
 do_pre_regen() {
   pending_dir=$1
 
+  declare -a ip_yuno=(ip.yunohost.org 0-ip.yunohost.org)
+  declare -a ip_yuno6=(ip.yunohost.org 0-ip6.yunohost.org)
```

Comment:
```suggestion
  declare -a ip_yuno6=(ip6.yunohost.org 0-ip6.yunohost.org)
```

Reply:
Indeed, thanks. I make it draft because I didn't check it (test). :-)